src/runtime.js

```diff
@@ -119,6 +120,7 @@ export function installRuntimeServices(global) {
   installTimerService(global);
   installVsyncService(global);
   installXhrService(global);
+  installCachedXhrService(global);
```

Comment:
Here are the correct numbers:

| | max | min | gzip | file |
|--|--|--|--|--|
**master** |  1.11 MB   |    206.7 kB   |                 61.94 kB   |   v0.js / amp.js
 **PR #7562** | 1.11 MB   |   207.05 kB   |                              62.11 kB   |   v0.js / amp.js